src/main/java/com/nvidia/spark/rapids/jni/kudo/KudoSerializer.java

```diff
@@ -172,6 +172,7 @@ public class KudoSerializer {
 
   public KudoSerializer(Schema schema) {
     requireNonNull(schema, "schema is null");
+    ensure(schema.getNumChildren() > 0, "Top schema can't be empty");
```

Comment:
So if we are going to write out a rows only batch we cannot do it with this API?